visualization/bno-world-svg/bno-world-csv2svg

```diff
@@ -43,7 +43,7 @@ $css_append = '';
 # line-wise
 foreach($lines as $line) {
  # first, discard comments
- if($line[0] != '#') {
+ if(isset($line[0]) and $line[0] != '#') {
```

Comment:
I know some people have severe PHP configurations, I would prefer to just explicitly resolve the PHP configuration at the start of the file instead of adding syntactic workarounds on every operation.

Reply:
Noted. I will remove it.